### ISSUE TITLE
Training Script and Model Implementation

### ISSUE THREAD

**Issue by nickbhat** [gussow/seeker#3]
Hi,

Do you plan to add a script for downloading data and retraining the model to this repository? This aids considerably in reproducing and interpreting results. 

Much appreciated!

**Comment by gussow** [comment]
Hi! Our accompanying paper has the information you are looking for - the supplementary data includes the samples used for training, which can be downloaded from NCBI (they are all publicly available). These should be straightforward to download, but if you have trouble with this, feel free to email us.

**Comment by nickbhat** [comment]
A few more questions re training -

1. What were your adam hyperparameters? Did you use any learning rate schedules?
2. Did you feed in the final hidden state to the classification layer, or some function of all hidden states?
